scripts/sequence-properties.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const fs = require('fs');
+const regenerate = require('regenerate');
+
+const regenerateToCode = require('./utils/regenerate-to-code');
+
+const basePath = `${__dirname}/../data/sequence-properties`;
+
+const sequencePropertyNames = [
+	'Basic_Emoji',
+	'Emoji_Keycap_Sequence',
+	'RGI_Emoji_Modifier_Sequence',
+	'RGI_Emoji_Flag_Sequence',
+	'RGI_Emoji_Tag_Sequence',
+	'RGI_Emoji_ZWJ_Sequence',
+	'RGI_Emoji',
+];
+
+fs.writeFileSync(
+	`${basePath}.js`,
+	`module.exports = new Set(${JSON.stringify(sequencePropertyNames, null, 2)});`
```

Comment:
Could also use `jsesc` (here and below) for more control over the output (e.g. if we want the string literals to be wrapped in single quotes). I believe that's what we do for `regenerate-unicode-properties` as well.